printable-recipe-generator/src/assets/recipe.ejs

```diff
@@ -266,6 +280,17 @@
         <% } %>
     </div>
 </footer>
+
+<%if (qrImageDataUrl && qrImageDataUrl!=null || qrImageDataUrl?.length>0) { %>>
+
+<div class="qr-section">
+    <p>📱 Open in App</p>
+    <img src="<%= qrImageDataUrl %>" alt="Scan to open app" />
+    <p>feastbraze://recipe/<%= recipe.id %></p>
+    <p>or visit: <a href="https://guardian-feast.go.link/recipe/<%= recipe.id %>">Recipe Here</a></p>
```

Comment:
this is pointless if you're printing it out! :grin:

Reply:
Yeah I will be removing these extras 😁